cts/cli/regression.tools.exp

```diff
@@ -1108,8 +1108,8 @@ Error performing operation: Node not found
 Current cluster status:
 Online: [ node1 ]
 
- dummy	(ocf::pacemaker:Dummy):	Stopped
- Fence	(stonith:fence_true):	Stopped
+ dummy	(ocf::pacemaker:Dummy):	 Stopped
```

Comment:
These extra spaces are from something @clumens added IIRC. They shouldn't make a difference since CTS's diff ignores whitespace. I figured I'd go ahead and add this .exp file with the others. Let me know if you'd rather I remove it.

Reply:
This is fine